src/components/GiveProject/GrantCard.tsx

```diff
@@ -96,6 +96,8 @@ export default function GrantCard({ grant, mode }: GrantDetailsProps) {
 
   const svgFillColour: string = theme.palette.type === "light" ? "black" : "white";
 
+  // Resets the viewport to the top of the page when pathnames change rather than
+  // preserving vertical position of the page you are coming from
   useEffect(() => {
     const items = document.getElementsByClassName("project-container");
```

Comment:
We're doing away with all of the custom classes, so it's better that you use an `id` attribute to refer to that particular element.

Reply:
changed, but not sure if it's what you meant so probably would be good to check it again when I re-push